frontend/api/v1_1/routes.py

```diff
@@ -46,7 +44,17 @@ def define_routes(application):
     # tags routes
     application.route("/tags",
                       callback=tags.list_available_tags)
+    # files routes
+    application.route("/files",
+                      callback=files.list)
+    # files routes
+    application.route("/files/<sha256>/download",
+                      callback=files.download)
+    application.route("/files/<sha256>/results",
+                      callback=files.results)
+    application.route("/files/<sha256>/scans",
```

Comment:
I was thinking that's it's possible to merge `/files/<sha256>/results` and `/files/<sha256>/scans`. In fact, the `file_web_schema` object, should have the reference to the scan, so it will be possible for a user, using a `/results` route, to link to a `/scans/<scanid>` and then get all informations from the scans.

So, I propose to include in the `file_web_schema` object, a `"scan": <scanid>` attribute, and remove the `/files/<sha256>/scans` route